api/models/app.go

```diff
@@ -59,7 +59,32 @@ func ListApps() (Apps, error) {
 }
 
 func GetApp(name string) (*App, error) {
-	res, err := DescribeStack(name)
+	return GetAppFast(name, true, true)
```

Comment:
GetApp() is a really crucial function, so I'd prefer that it still inlines the logic to prepend the rack name, describe the stack, etc.

GetAppUnbound() or GetAppLegacy() can duplicate code too.

After all, the long term goal is to return to a single path, which can be as simple as removing the GetAppUnbound() function.